doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md

```diff
@@ -21,7 +21,7 @@ And we also need two corresponding test classes:
     And one more **package for tests**:  
     `/virtual-schemas/src/test/java/com/exasol/adapter/dialects/<your new dialect package>` 
 
-    For example, the package for Athena is named `com.exasol.adapter.dialects.athena`.
+    For example, the package for Athena has a name `com.exasol.adapter.dialects.athena`.
```

Comment:
Please revert this change. This sounds strange.

Reply:
done